plugins/inputs/webhooks/particle/README.md

```diff
@@ -0,0 +1,35 @@
+# particle webhooks
+
+You should configure your Particle.io's Webhooks to point at the `webhooks` service. To do this go to `console.particle.io/` and click `Integrations > New Integration > Webhook`. In the resulting page set `URL` to `http://<my_ip>:1619/particle`, and  under `Advanced Settings` click on `JSON` and add:
```

Comment:
Turn `console.particle.io/` into a url: `[name of link](https://console.particle.io/)`